test/integration/manifests/cns/daemonset-windows.yaml

```diff
@@ -0,0 +1,134 @@
+    spec:
+      securityContext:
+        windowsOptions:
+          hostProcess: true
+          runAsUserName: "NT AUTHORITY\\SYSTEM"
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: kubernetes.azure.com/cluster
+                    operator: Exists
+                  - key: type
+                    operator: NotIn
+                    values:
+                      - virtual-kubelet
+                  - key: kubernetes.io/os
+                    operator: In
+                    values:
+                      - windows
+      tolerations:
+        - operator: "Exists"
+          effect: NoExecute
+        - operator: "Exists"
+          effect: NoSchedule
+      containers:
+        - name: cns-container
+          image: acnpublic.azurecr.io/azure-cns:latest
+          imagePullPolicy: IfNotPresent
+          securityContext:
+            privileged: true
+          command: ["powershell.exe"]
+          args:
+            [
+              '.\setkubeconfigpath.ps1',
+              ";",
+              '.\azure-cns.exe',
+              "-c",
+              "tcp://$(CNSIpAddress):$(CNSPort)",
+              "-t",
+              "$(CNSLogTarget)",
+              "-o",
+              "$(CNSLogDir)",
+              "-storefilepath",
+              "$(CNSStoreFilePath)",
+              "-config-path",
+              "%CONTAINER_SANDBOX_MOUNT_POINT%\\$(CNS_CONFIGURATION_PATH)",
+              "--kubeconfig",
+              '.\kubeconfig',
+            ]
+          volumeMounts:
+            - name: log
+              mountPath: /k/azurecns
+            - name: cns-config
+              mountPath: /etc/azure-cns
+            - name: cni-conflist
+              mountPath: /k/azurecni/netconf
+          ports:
+            - containerPort: 10090
+              hostPort: 10090
+              name: api
+              protocol: TCP
+            - containerPort: 10092
+              hostPort: 10092
+              name: metrics
+              protocol: TCP
+          env:
+            - name: PATH
+              value: '%CONTAINER_SANDBOX_MOUNT_POINT%\Windows\System32\WindowsPowershell\v1.0\'
+            - name: CNSIpAddress
+              value: "127.0.0.1"
+            - name: CNSPort
+              value: "10090"
+            - name: CNSLogTarget
+              value: "stdoutfile"
+            - name: CNSLogDir
+              value: /k/azurecns/
+            - name: CNSStoreFilePath
+              value: /k/azurecns/
+            - name: CNS_CONFIGURATION_PATH
+              value: /etc/azure-cns/cns_config.json
+            - name: NODENAME
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: spec.nodeName
+      initContainers:
+        - name: init-cni-dropgz
+          image: acnpublic.azurecr.io/cni-dropgz:latest
```

Comment:
Do we publish the `latest` tag for cni-dropgz ?

Reply:
No we don't, but this is just a placeholder, we update it in code through `CNI_DROPGZ_VERSION`
I'll update this here too to reflect that